client.py

```diff
@@ -34,6 +23,67 @@
         if iteration_counter == 10:
             iteration_counter = 0
             print("Map size:", my_map.size())
+
+
+def sql_example(hz_client):
+    print("Creating a mapping...")
+    # See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
+    mapping_query = "CREATE MAPPING cities TYPE IMap OPTIONS ('keyFormat'='varchar','valueFormat'='varchar')"
+    hz_client.sql.execute(mapping_query)
```

Comment:
```suggestion
    hz_client.sql.execute(mapping_query).result()
```
I think it would be best to wait for the execution of the CREAT MAPPING before moving into other commands

Reply:
Fixed